components/match2/commons/match_group_coordinates.lua

```diff
@@ -289,4 +289,69 @@ function MatchGroupCoordinates.groupMatchIdsByField(bracket, fieldName)
 	return byField
 end
 
+--[[
+Compute the number of opponents from each round onward in a bracket.
```

Comment:
Am I correct in understanding this is 
`Computes an array, where each key is round (column) of the bracket, and the value is the number of opponent who can still win/who are not yet eliminated`?

Reply:
`Computes an array, where each key is round (column) of the bracket, and the value is the number of opponents who are either in the current round (column) or are seeded into a later round (column).`

(they can theoretically already have lost in the round we view, but they still were in the round)

also if we have a DE bracket it considers stuff correctly too

so as a tldr it computes the numbers of players still present in the tournament/bracket at a given round

Reply:
adjusted the comment